docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -0,0 +1,267 @@
+
+</dl>
+
+## Prerequisites
+
+Before setting up your UID2 Operator service in the Google Cloud Platform using Confidential Space, complete these prerequisite steps:
+
+- [Confidential Space Account Setup](#confidential-space-account-setup).
+- [UID2 Operator Account Setup](#uid2-operator-account-setup).
+
+### Confidential Space Account Setup
+
+You can run the UID2 Operator service on any GCP account and project. However, to support attestation, you'll need to create a service account that you can use to run Confidential Space virtual machines (VMs), and then grant it specific permissions.
+
+to set up and configure the account, go to [Google Cloud Console](https://console.cloud.google.com/) and complete the following steps.
+
+These steps use placeholder values for the following: `PROJECT_ID`, `SERVICE_ACCOUNT_NAME`. (**GWH_YS02 where do they get these values from? Is it just their choice**)
+
+1. Click **Active Cloud shell**.
+
+2. Switch to your project:
+    ```
+    $ gcloud config set project {PROJECT_ID}
+    ```
+ 
+3. Enable required APIs:
+    ```
+    $ gcloud services enable compute.googleapis.com confidentialcomputing.googleapis.com
+    ```
+
+4. Create a service account to run the <a href="#gl-workload">workload</a>:
+    ```
+    $ gcloud iam service-accounts create {SERVICE_ACCOUNT_NAME}
+    ```
+
+5. Grant the required permission, `confidentialcomputing.workloadUser`, to the service account. This permission grants the ability to generate an attestation token and run a workload in a VM:
+    ```
+    $ gcloud projects add-iam-policy-binding {PROJECT_ID} \
+      --member=serviceAccount:{SERVICE_ACCOUNT_NAME}@{PROJECT_ID}.iam.gserviceaccount.com \
+      --role=roles/confidentialcomputing.workloadUser
+    ```
+
+6. (Optional) Grant the optional permission, `logging.logWriter`, to the service account. This permission grants access to write and view logs in debug mode, and is useful for testing:
+    ```
+    $ gcloud projects add-iam-policy-binding {PROJECT_ID} \
+      --member=serviceAccount:{SERVICE_ACCOUNT_NAME}@{PROJECT_ID}.iam.gserviceaccount.com \
+      --role=roles/logging.logWriter
+    ```
+  
+7. Add a VPC rule to allow public access on port 8080, the default exposed port for the UID2 operator:
+    ```
+    $ gcloud compute firewall-rules create operator-tcp \
+      --direction=INGRESS --priority=1000 --network=default --action=ALLOW \
+      --rules=tcp:8080 \
+      --source-ranges=0.0.0.0/0 \
+      --target-service-accounts={SERVICE_ACCOUNT_NAME}@{PROJECT_ID}.iam.gserviceaccount.com
+    ```
+
+### UID2 Operator Account Setup
+Ask your UID2 contact to register your organization as a UID2 Operator. If you're not sure who to ask, see [Contact Info](../getting-started/gs-account-setup.md#contact-info).
+
+(**GWH_YS03 what information do they need to provide for registration? In case you don't know, who should I ask?**)
```

Comment:
I don't know. Could ask kishan.khatri or andrei.tarassov
.

Reply:
Added that info -- thx.